test/config.tests.js

```diff
@@ -51,6 +51,20 @@ describe('config', function() {
     });
   });
 
+  describe('when authorizationParams response_type fuzzy matches issuer', function() {
+    const customConfig = Object.assign({}, defaultConfig, {
+      clientSecret: '__test_client_secret__',
+      authorizationParams: {
+        response_type: 'token id_token code'
+      }
+    });
+    const config = getConfig(customConfig);
+
+    it('should keep token code', function() {
+      assert.equal(config.authorizationParams.response_type, 'token id_token code');
```

Comment:
Does not exactly match [metadata fixture](https://github.com/auth0/express-openid-connect/blob/master/test/fixture/well-known.json#L12) so this would have errored before